docs/data_collection.rst

```diff
@@ -121,49 +132,14 @@ An impression is whenever a user is shown a specific snippet.
 Snippet Clicks
 ~~~~~~~~~~~~~~
 
-Whenever a link in a snippet is clicked, we trigger an event that notes which
-particular link was clicked. This includes links that may trigger an action
-besides opening up a new page, such as links that trigger browser menus.
+Whenever a link in a snippet is clicked or a form is submitted, we trigger an event
+that notes which particular link was clicked. This includes links that may trigger
+an action besides opening up a new page, such as links that trigger browser menus,
+as well as buttons that submit a form.
```

Comment:
Can we make this more generic to include all interactions with snippets, like moving from scene 1 to scene 2 on some templates.